packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/process-node.js

```diff
@@ -861,6 +891,7 @@ const processNodeString = async ({
     replaceNodeHtmlImages,
     replaceFileLinks,
     replaceNodeHtmlLinks,
+    searchAndReplaceNodeStrings,
```

Comment:
Let's move this to the top of the array. These are run in order one at a time and if we run it at the end we'll have already done a bunch of image/file processing and manipulation of the node data. Putting this at the end means we wont be able to rewrite our WP URL to a CDN url before Gatsby pulls all the images in from WP.